### PR TITLE
fix showcase app

### LINKED ISSUE #204: struts2-jquery-plugin not compatible with struts 6.0.0
Hello, today I updated struts from 2.5.30 to 6.0.0.

My app compiles and starts fine but when I try to access a page I get the following error:
```
o.a.struts2.components.ClosingUIBean:57 - Could not open template
freemarker.core.ParseException: Syntax error in template "template/jquery/autocompleter.ftl" in line 30, column 47:
Using ?html (legacy escaping) is not allowed when auto-escaping is on with a markup output format (HTML), to avoid double-escaping mistakes.
```

I tried setting the flag `struts.ui.escapeHtmlBody` to `false` but this did not solve the problem. Installing version 4.1.0-SNAPSHOT did not help either.
Do you know if it would somehow be possible to use the plugin with the newest struts? Maybe there is some other flag I overlooked?

When I add the property `<struts2.version>6.0.0</struts2.version>` to the pom of `struts2-jquery-integration-tests` and run `mvn verify` I get lots of similar errors, for example:
```
Caused by: freemarker.core.ParseException: Syntax error in template "template/jquery/div.ftl" in line 22, column 43:
Using ?html (legacy escaping) is not allowed when auto-escaping is on with a markup output format (HTML), to avoid double-escaping mistakes.
```

### LINKED ISSUE #218: get showcase app working again

### LINKED ISSUE #222: fix Link with Highlight Effect in showcase-app
```
VM331:10 Uncaught SyntaxError: Unexpected token '&'
    at b (jquery-3.6.0.min.js:2:866)
    at He (jquery-3.6.0.min.js:2:48373)
    at S.fn.init.append (jquery-3.6.0.min.js:2:49724)
    at S.fn.init.<anonymous> (jquery-3.6.0.min.js:2:50816)
    at $ (jquery-3.6.0.min.js:2:32425)
    at S.fn.init.html (jquery-3.6.0.min.js:2:50494)
    at Object.success (jquery.struts2.min.js?s2j=5.0.0-SNAPSHOT:26:99)
    at c (jquery-3.6.0.min.js:2:28327)
    at Object.fireWith [as resolveWith] (jquery-3.6.0.min.js:2:29072)
    at l (jquery-3.6.0.min.js:2:79901)
```

### LINKED ISSUE #223: fix div - Resizable in showcase-app
```
VM348:1 Uncaught SyntaxError: Unexpected token '&'
    at Object.resizable (<anonymous>:26:62)
    at Object.complete (jquery.struts2.min.js?s2j=5.0.0-SNAPSHOT:29:164)
    at c (jquery-3.6.0.min.js:2:28327)
    at Object.fireWith (jquery-3.6.0.min.js:2:29072)
    at l (jquery-3.6.0.min.js:2:80029)
    at XMLHttpRequest.<anonymous> (jquery-3.6.0.min.js:2:82355)
```

### LINKED ISSUE #225: fix div > Drag and Drop in showcase-app
```
jquery-3.6.0.min.js:2 Uncaught SyntaxError: Unexpected token '&'
    at Object.draggable (<anonymous>:19:421)
    at Object.container (jquery.struts2.min.js?s2j=5.0.0-SNAPSHOT:36:133)
    at Object.bind (jquery.struts2.min.js?s2j=5.0.0-SNAPSHOT:30:215)
    at HTMLDocument.<anonymous> (<anonymous>:9:23)
    at e (jquery-3.6.0.min.js:2:30038)
    at t (jquery-3.6.0.min.js:2:30340)
```

```
jquery-3.6.0.min.js:2 Uncaught SyntaxError: Unexpected token '&'
    at Object.droppable (<anonymous>:21:109)
    at Object.container (jquery.struts2.min.js?s2j=5.0.0-SNAPSHOT:36:166)
    at Object.bind (jquery.struts2.min.js?s2j=5.0.0-SNAPSHOT:30:215)
    at HTMLDocument.<anonymous> (<anonymous>:9:23)
    at e (jquery-3.6.0.min.js:2:30038)
    at t (jquery-3.6.0.min.js:2:30340)
```

### PR DESCRIPTION
**Pull request by sdutry**
issue
- #218 
- #222
- #223
- #225

remaining problems have a seperate issue created:
- #224
- #226
- #227
- #228 

Changes done:
- load static files from under `/static`
- remove `?html` from ftl templates, further checking will be needed to see if there is need for escaping in some cases and how to solve this.

### CODE REVIEW

File: struts2-jquery-plugin/src/main/resources/template/jquery/autocompleter-close.ftl [hunk 1]
```diff
@@ -22,49 +22,49 @@
 <script type='text/javascript'>
 jQuery(document).ready(function () {
   <#if parameters.valueWidget?if_exists != "">
-	jQuery("#${parameters.id?html}").val("${parameters.valueWidget?html}");
+	jQuery("#${parameters.i}").val("${parameters.valueWidget}");
```
Comment (lukaszlenart) [inline struts2-jquery-plugin/src/main/resources/template/jquery/autocompleter-close.ftl]: `i`? shouldn't be `id`?

Comment (sdutry) [reply struts2-jquery-plugin/src/main/resources/template/jquery/autocompleter-close.ftl]: Will look into it when i get home, probably already fixed it localy and forgot to commit it.

Comment (sdutry) [reply struts2-jquery-plugin/src/main/resources/template/jquery/autocompleter-close.ftl]: fixed

**Comment by sdutry** [comment]
added some fixes for the issues created seperately

**Review by lukaszlenart — APPROVED** [review]
LGTM 👍

**Pull request merged**